docker-entrypoint.sh

```diff
@@ -1,22 +1,46 @@
 #!/bin/bash
 set -eo pipefail
 
+# For reset or first database setup ONLY!!!
```

Comment:
What does "first" mean here?  Can you clarify this a bit.  Could you say:

> For initial setup, or to completely recreate the database. ALL DATA WILL BE LOST!

Reply:
Yes, you understand correctly. I was talking about initial setup.